noir-projects/noir-protocol-circuits/bootstrap.sh

```diff
@@ -65,7 +65,12 @@ function compile {
       echo "Error: $json_path bytecode size of $bytecode_size exceeds 850MB"
       exit 1
     fi
-    cache_upload circuit-$hash.tar.gz $json_path &> /dev/null
+    # Disable cache upload on ARM when AVM is disabled to prevent caching wrong keys
+    if [[ $(arch) == "arm64" && "${DISABLE_AZTEC_VM:-0}" -eq 1 ]]; then
```

Comment:
I rather change the cache hash here. Will disabled Aztec vm be set here? I think not